packages/framework/presence/src/presence.ts

```diff
@@ -56,8 +56,8 @@ export type SessionClientStatus =
  *
  * `ISessionClient` should be used as key to distinguish between different
  * clients as they join, rejoin, and disconnect from a session. While a
- * client's {@link ClientConnectionId} may change over time `ISessionClient`
- * will be fixed.
+ * client's {@link ClientConnectionId} read from {@link ISessionClient.getConnectionStatus}
+ * may change over time `ISessionClient` will be fixed.
```

Comment:
```suggestion
 * client's {@link ClientConnectionId} from {@link ISessionClient.getConnectionStatus}
 * may change over time, `ISessionClient` will be fixed.
```

Reply:
Funny. I have this pending but decided no to push until I saw if there was anymore feedback. :)